settings.php

```diff
@@ -1067,10 +1067,10 @@ function convertseconds($argument)
                                                     <div class="radio">
                                                         <label><input type="radio" name="privacylevel" value="4"
                                                                       <?php if ($privacylevel === 4){ ?>checked<?php }
-                                                            ?>>No Statistics mode: This disables all statistics processing. Even the query counters will not be available.<br>Additionally, you can disable logging to the file <code>/var/log/pihole.log</code> using <code>sudo pihole logging off</code>.</label>
+                                                            ?>>No Statistics mode: This disables all statistics processing. Even the query counters will not be available.<br><strong>Note that regex blocking is not available when query analyzing is disabled.</strong><br>Additionally, you can disable logging to the file <code>/var/log/pihole.log</code> using <code>sudo pihole logging off</code>.</label>
                                                     </div>
                                                 </div>
-                                                <p>The privacy level may be changed at any time without having to restart the DNS resolver. However, note that queries with (partially) hidden details cannot be disclosed with a subsequent reduction of the privacy level.</p>
+                                                <p>The privacy level may be increased at any time without having to restart the DNS resolver. However, note that you need to restart the DNS resolver when lowering the privacy level.</p>
```

Comment:
Could the case of lowering the privacy level be detected when saving, and then FTL would be automatically restarted? (This would happen in `savesettings.php`)

Reply:
Privacy level 1 -> 2
![screenshot from 2018-11-10 08-38-06](https://user-images.githubusercontent.com/16748619/48298832-5d9b9380-e4c4-11e8-8ff4-069bfddcf3aa.png)

Privacy level 2 -> 0
![screenshot from 2018-11-10 08-38-21](https://user-images.githubusercontent.com/16748619/48298835-69875580-e4c4-11e8-8f55-199f89394235.png)

Privacy level 1 -> 1
![screenshot from 2018-11-10 08-41-39](https://user-images.githubusercontent.com/16748619/48298836-773cdb00-e4c4-11e8-894c-71aa57c21fe1.png)